src/tests/test_utils/common_test_utils/src/data_utils.cpp

```diff
@@ -301,6 +304,50 @@ void fill_psroi(ov::Tensor& tensor,
     }
 }
 
+void fill_psroi(ov::Tensor& tensor,
+                int batchSize,
+                int height,
+                int width,
+                int groupSize,
+                float spatialScale,
+                int spatialBinsX,
+                int spatialBinsY,
+                const std::string& mode) {
+#define CASE(X)                                        \
+    case X:                                            \
+        fill_psroi_impl<X>(tensor,                     \
+                           batchSize,                  \
+                           height,                     \
+                           width,                      \
+                           groupSize,                  \
+                           spatialScale,               \
+                           spatialBinsX,               \
+                           spatialBinsY,               \
+                           mode); break;
+
+    auto element_type = tensor.get_element_type();
+    switch (element_type) {
+        CASE(ov::element::f64)
+        CASE(ov::element::f32)
+        CASE(ov::element::f16)
+        CASE(ov::element::bf16)
+        CASE(ov::element::u1)
+        CASE(ov::element::u4)
+        CASE(ov::element::u8)
+        CASE(ov::element::u32)
+        CASE(ov::element::u16)
```

Comment:
The most part of these data types are prohibited and cause exception in `generate(const std::shared_ptr<ov::op::v0::PSROIPooling>& node..)` function, so why do we need them here?